README.md

```diff
@@ -82,22 +82,58 @@ Navigate to http://localhost:8080/opsui, you should see the default OPSUI system
 
 To contribute a patch, follow these instructions (note that installing
 [Hub](http://hub.github.com) is not strictly required, but is recommended).
+```
+Download and install hub.github.com
```

Comment:
We can remove this line since this is optional and no need to be highlighted anymore even though it was there previously.

Reply:
Fixed as suggested.